devices/legrand.js

```diff
@@ -28,6 +28,29 @@ const tzLocal = {
 };
 
 module.exports = [
+    {
+        zigbeeModel: [' Pocket remote\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'+
+            '\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'],
+        fingerprint: [{modelID: 'GreenPower_254', ieeeAddr: /^0x00000000005.....$/}],
```

Comment:
Either the fingerprint or zigbeeModel should not be needed, can you try without the fingerprint?

Reply:
It's working. However I've got this error message, (I have it with "fingerprint" too)) : 

'''
Failed to configure '0x000474000092b288', attempt 1 (TypeError: Cannot read properties of undefined (reading 'bind') at Object.bind (/app/node_modules/zigbee-herdsman-converters/lib/reporting.js:40:24) at Object.configure (/app/data/extension/externally-loaded.js:34:29) at Configure.configure (/app/lib/extension/configure.ts:117:13))
'''

Reply:
Should be fixed after removing these two lines:

```
            const endpoint2 = device.getEndpoint(2);
             await reporting.bind(endpoint2, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
```

If not, provide me the data/database.db entry of this device.

Reply:
It's working.
However I am unable to use the device in any automation.
As a matter of fact it has 4 button and here is what is logged when I press each of them

```
Debug 2023-01-28 12:41:16Received Zigbee message from '0x000474000092b288', type 'commandRecall', cluster 'genScenes', data '{"groupid":65514,"sceneid":1}' from endpoint 1 with groupID 65514
Info 2023-01-28 12:41:16MQTT publish: topic 'zigbee2mqtt/0x000474000092b288', payload '{"action":null,"battery":20,"last_seen":1674906076902,"linkquality":174,"voltage":2600}'
Debug 2023-01-28 12:41:19Received Zigbee message from '0x000474000092b288', type 'commandRecall', cluster 'genScenes', data '{"groupid":65516,"sceneid":1}' from endpoint 1 with groupID 65516
Info 2023-01-28 12:41:19MQTT publish: topic 'zigbee2mqtt/0x000474000092b288', payload '{"action":null,"battery":20,"last_seen":1674906079400,"linkquality":25,"voltage":2600}'
Debug 2023-01-28 12:41:21Received Zigbee message from '0x000474000092b288', type 'commandRecall', cluster 'genScenes', data '{"groupid":65517,"sceneid":1}' from endpoint 1 with groupID 65517
Info 2023-01-28 12:41:21MQTT publish: topic 'zigbee2mqtt/0x000474000092b288', payload '{"action":null,"battery":20,"last_seen":1674906081124,"linkquality":102,"voltage":2600}'
Debug 2023-01-28 12:41:22Received Zigbee message from '0x000474000092b288', type 'commandRecall', cluster 'genScenes', data '{"groupid":65515,"sceneid":1}' from endpoint 1 with groupID 65515
Info 2023-01-28 12:41:22MQTT publish: topic 'zigbee2mqtt/0x000474000092b288', payload '{"action":null,"battery":20,"last_seen":1674906082212,"linkquality":40,"voltage":2600}'
```
The value of groupID is modified (4 possible values since 4 buttons), but I don't manage to catch it in an automation

Reply:
Try adding `fz.command_recall` to `fromZigbee`

Reply:
Thanks, it's working now !

Reply:
Unfortunately I am unable to get it works.
I only have this in the state

```
action: ''
battery: 40
last_seen: 1675113110530
linkquality: 32
voltage: 2700
icon: mdi:gesture-double-tap
friendly_name: legrand_switch_scenario_action
```
the "action_group" doesn't appear, and I didn't find a way to use it despite several tests. Do you have any idea ?

Reply:
The action group will be cleared immediately, (since this is an action not a state, you can use https://www.zigbee2mqtt.io/guide/usage/integrations/home_assistant.html#via-mqtt)

Reply:
Unfortunately I have this, and it doesn't work. (It works without a payload... but then I am unable to determine which button was used)

```
alias: "[SCENARIO] Legrand 1"
description: ""
trigger:
  - platform: mqtt
    topic: zigbee2mqtt/legrand_switch_scenario
    payload: "65517"
action:
  - service: light.toggle
    data: {}
    target:
      entity_id: light.g_bureau_mur
mode: single
```

Reply:
> This looks good, see https://www.zigbee2mqtt.io/guide/usage/integrations/home_assistant.html#responding-to-button-clicks how to use it in your HA automation.

@Koenkk if you have a moment could you look at https://github.com/Koenkk/zigbee2mqtt/issues/16170 and tell me if you have any ideas? I'm unable to create any sort of automation with this device; I only have "when battery level changes" as a trigger. thank you